modules/ose_3_0_release_notes.adoc

```diff
@@ -262,7 +262,7 @@ there.
 It is not possible to do correct JNDI lookup for datasources in the current JWS
 image if an invalid combination of datasource and realm properties is defined.
 If a datasource is configured in the *_context.xml_* file and a realm in the
-*_server.xml_* file, then the *_server.xml_* file's `*localDataSource*` property
+*_server.xml_* file, then the *_server.xml_* file's `localDataSource` property
```

Comment:
I'd revert the changes to file. It seems to be about 3.x and is using the correct styles for it.

Reply:
@kalexand-rh, will do, thanks!